backend/app/utilities/firebase_init.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+import firebase_admin
+from firebase_admin import credentials
+
+
+def initialize_firebase():
```

Comment:
add INFO level logs for starting and successful initialization of firebase.

Reply:
Not working, so creating a [ticket](https://www.notion.so/uwblueprintexecs/Firebase-Initialization-Logging-13f10f3fb1dc8062b95fd6194988a08c?pvs=4) for this to push it.